transmart-rest-api/build.gradle

```diff
@@ -35,8 +35,11 @@ dependencies {
     testCompile "org.grails:grails-plugin-testing"
     testRuntime "net.sourceforge.htmlunit:htmlunit:2.18"
     testCompile 'org.grails:grails-datastore-rest-client'
+    compile 'org.quartz-scheduler:quartz:2.2.1'
+    compile 'net.sf.opencsv:opencsv:2.3'
 
     compile 'org.grails.plugins:spring-security-core:3.1.1'
+    compile project(':transmart-legacy-db')
```

Comment:
We would better to duplicate them in the core-db and remove this dependency. See `*CoreDb.groovy` domain classes in core-db.